### PR TITLE
Add URI Handlers

### PR DESCRIPTION
**Pull request by emma-miler**

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
(Will need localisations for other languages when merged)

**Comment by abarichello** [comment]
image
you literally have to run one command

**Comment by emma-miler** [comment]
counterargument: no

**Comment by GeckoEidechse** [comment]
Requires https://github.com/R2Northstar/NorthstarMods/pull/213

**Comment by emma-miler** [comment]
Seems like this broke somewhere along the way. Give me some time to fix before merging ^^

File: LauncherInjector/main.cpp [hunk 1]
```diff
@@ -250,105 +311,155 @@ int main(int argc, char* argv[])
...
+		return 1;
+	}
 
-	for (int i = 0; i < argc; i++)
-		if (!strcmp(argv[i], "-noOriginStartup"))
-			noOriginStartup = true;
-		else if (!strcmp(argv[i], "-dedicated")) // also checked by Northstar.dll
-			dedicated = true;
-		else if (!strcmp(argv[i], "-nostubs"))
-			nostubs = true;
```
Comment (pg9182) [inline LauncherInjector/main.cpp:263]: Another merge issue...

**Comment by emma-miler** [comment]
Pushing this back to 1.7.0, will rebase on 1.6.0 once it comes out.

**Pull request closed without merging**